### DISCUSSION TITLE
Better npm search proposal

### DISCUSSION THREAD

**Discussion by astoilkov** [community/community#128439] [npm]
[Better npm search proposal](https://astoilkov.com/better-npm-search-proposal) is my exploration of how the npm search can be improved.

The main idea is to give fewer points for popularity and more points for consistency of commits, releases, and issue responses. The longer the period of consistency, the better. The article is an in-depth exploration of the idea.

**Comment by monishcm** [comment]
Thanks for your feedback and wonderful write up. We are investigating on how to improve the search experience in npm and make it simpler to use.
